### PR TITLE
build: automatically generate luvref.txt with panvimdoc

### PR DESCRIPTION
**Pull request by dundargoc**

### CODE REVIEW

File: runtime/doc/luvref.txt [hunk 1]
```diff
@@ -1,36 +1,24 @@
...
-is exposed as `vim.uv.version()`).
-
-For information about this manual, see |luv-credits|.
-
-For further examples, see https://github.com/luvit/luv/tree/master/examples.
+*luvref.txt*                 For                 Last change: 2023 November 24
 
 ==============================================================================
-INTRODUCTION                                                  *luv* *luv-intro* *uv*
+1. LibUV in Lua                                          *luvref-libuv-in-lua*
```
Comment (clason) [inline runtime/doc/luvref.txt:4]: Is there any way of preserving the current tags? If not, that would be kind of a deal-breaker...

Comment (dundargoc) [reply runtime/doc/luvref.txt]: I don't think so, I think the latest change is as close as we'll get at the moment.

Comment (clason) [reply runtime/doc/luvref.txt]: Then I'm afraid that's a -1 from me...

Comment (dundargoc) [reply runtime/doc/luvref.txt]: Yeah kinda agree.

**Comment by dundargoc** [comment]
The changes from panvimdoc are too intrusive to go ahead with this. It would be nice to get this to work, but not a high priority. If anyone wants to make the necessary fixes upstream to get this to work then and continue this PR then that would of course be welcome.

**Pull request closed without merging**